### DISCUSSION TITLE
How do I enable Fluentbit logs in a ECS Fargate container

### DISCUSSION THREAD

**Discussion by fsanzdev** [gruntwork-io/knowledge-base#304] [Help] [answered]
Hi, 

I'm looking to do something like this https://docs.aws.amazon.com/AmazonECS/latest/developerguide/firelens-example-taskdefs.html#firelens-example-forward in a Fargate service. I'm using the service example in the Ref Arch, but I can't find the proper way to add a new container to the task definition. 

Thanks.

**Comment by tonerdo** [comment, marked as answer]
@fsanzdev please take a look, there should be an `ecs_task_container_definitions` variable that allows you to specify the container definition json
